### PR TITLE
integrated sentry

### PR DESCRIPTION
**Pull request by Abhinav-grd**

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -1,19 +1,68 @@
...
     enabled: process.env.ANALYZE === 'true',
 });
 
+const {
+    NEXT_PUBLIC_SENTRY_DSN: SENTRY_DSN,
+    SENTRY_ORG,
+    SENTRY_PROJECT,
+    SENTRY_AUTH_TOKEN,
+    NODE_ENV,
+    VERCEL_GITHUB_COMMIT_SHA: COMMIT_SHA,
```
Comment (pushkar8723) [inline next.config.js:14]: Will `COMMIT_SHA` be present in the env variable during build time on cloud flare?

**Review by pushkar8723 — APPROVED** [review]

File: next.config.js (hunk 1 above)
Comment (Abhinav-grd) [reply next.config.js:14]: @vishnukvmd  Is there some way to get this.?

Comment (vishnukvmd) [reply next.config.js:14]: I don't see any. https://developers.cloudflare.com/pages/platform/build-configuration

Will require more digging.

For now we'll continue to use Netlify for bada-frame. They seem to have `COMMIT_REF`?

https://docs.netlify.com/configure-builds/environment-variables/

Comment (Abhinav-grd) [reply next.config.js:14]: Okk...then lets configure for netlify for now and leave the problem of configuring for cloudflare for when we migrate

**Pull request merged**